src/test/resources/application.properties

```diff
@@ -0,0 +1,16 @@
+# this is the test properties for default profile
+userservice.db.primary.url=jdbc:h2:mem:testdb;MODE=PostgreSQL
+userservice.db.primary.password=pass
+userservice.db.primary.driver=org.h2.Driver
+userservice.db.primary.hbm2ddl.auto=create-drop
+userservice.db.primary.user=dh
+userservice.db.primary.hibernate.dialect=org.hibernate.dialect.H2Dialect
```

Comment:
Since configs are not using userservice prefix anymore, the prefix can be removed.